operator/controllers/manifest_controller.go

```diff
@@ -380,8 +398,9 @@ func (r *ManifestReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Mana
 	failureBaseDelay time.Duration, failureMaxDelay time.Duration, frequency int, burst int, listenerAddr string,
 ) error {
 	r.DeployChan = make(chan OperationRequest, r.Workers.GetWorkerPoolSize())
+	r.ConsistencyCheckChan = make(chan ConsistencyCheckRequest, r.Workers.GetWorkerPoolSize())
```

Comment:
why not use one set of workers both for `processing` and `ready` states. Have 2 sets of workers is an overkill. Also if worker pool size if set to `5` for example, `10` workers will be started.

Reply:
I reverted code, consistence check workpool is gone.